plugin.video.orftvthek/resources/lib/helpers.py

```diff
@@ -15,12 +57,14 @@ def parameters_string_to_dict(parameters):
     return paramDict
 
 
-def debugLog(message,type):
-    output = "[ORF TVTHEK] ("+type+") "+message;
-    xbmc.log(msg=output, level=xbmc.LOGDEBUG)
+def debugLog(message, loglevel=xbmc.LOGDEBUG):
+    output = "[ORF TVTHEK] " + py2_encode(message)
+    #xbmc.log(msg=output, level=loglevel)
+
 
 def notifyUser(message):
     addon = xbmcaddon.Addon()
     name = addon.getAddonInfo('name')
     icon = addon.getAddonInfo('icon')
-    xbmc.executebuiltin('Notification(%s, %s, %s, %s)'%(name,message, "", icon))
+    xbmc.executebuiltin('Notification(%s, %s, %s, %s)' % (py2_encode(name), py2_encode(message), "", icon))
```

Comment:
`xbmcgui.Dialog().notification()` instead of the builtin please. It's supported since gotham.